Common/ac/game_version.h

```diff
@@ -138,7 +142,8 @@ enum GameDataVersion
     kGameVersion_341_2          = 49,
     kGameVersion_350            = 50,
     kGameVersion_351            = 51,
-    kGameVersion_Current        = kGameVersion_351
+    kGameVersion_399            = 60,
+    kGameVersion_Current        = kGameVersion_399
```

Comment:
The changes to game_version.h are unnecessary, because game data format itself was not modified.
That would be necessary if you had add these properties to the game data as well, so that they could be set in the editor and applied at startup.

Reply:
It was in my plans to add the property to property grid, what do you think?
If we do add it, do we also try to implement the blending modes in the editor preview?

Reply:
Well, my personal opinion is that properties should be same everywhere, as people may want to set up constant mode for object too. Affecting preview is an optional thing here imho, because editor is not perfectly synced with the engine anyway.